### PR TITLE
Upgrade to Django 3.2

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: basket/news/utils.py [hunk 1]
```diff
@@ -135,7 +135,7 @@ def has_valid_api_key(request):
         or request.POST.get("api_key", None)
         or request.GET.get("api-key", None)
         or request.GET.get("api_key", None)
-        or request.META.get("HTTP_X_API_KEY", None)
```
Comment (stevejalim) [inline basket/news/utils.py:138]: Did this work with the old header name? Looks like it might not have?

File: basket/news/views.py [hunk 2]
```diff
@@ -31,9 +30,7 @@
 from basket.news.models import (
     CommonVoiceUpdate,
     Interest,
-    LocaleStewards,
```
Comment (stevejalim) [inline basket/news/views.py:34]: Are these two unimported models still in use elsewhere? If not, would be good too throw them in the woodchipper while we're at it?

Comment (stevejalim) [inline basket/news/views.py:34]: Are these two unimported models still in use elsewhere? If not, would be good too throw them in the woodchipper while we're at it?

File: basket/settings.py [hunk 3]
```diff
@@ -64,7 +64,7 @@ def path(*args):
     DATABASES["default"]["OPTIONS"] = {
         "init_command": "SET sql_mode='STRICT_TRANS_TABLES'",
     }
-
+DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
```
Comment (stevejalim) [inline basket/settings.py:67]: Nice one. I would have forgotten to add this!

Comment (stevejalim) [inline basket/settings.py:67]: Nice one. I would have forgotten to add this!

**Review by stevejalim — APPROVED** [review]
Looks good to me! ❤️

File: basket/news/views.py (hunk 2 above)
Comment (robhudson) [reply basket/news/views.py:34]: Yeah the are still in use, just no longer needed to be used with the synctool here. Good check though.

**Review by pmac — APPROVED** [review]
🏅

**Pull request merged**